core/src/test/java/org/apache/iceberg/rest/TestRESTViewCatalog.java

```diff
@@ -244,6 +244,33 @@ public void viewExistsViaHEADRequest() {
             any());
   }
 
+  @Test
+  public void renameViewNamespaceHavingCatalogName() {
```

Comment:
```suggestion
  public void renameViewNamespaceWithCatalogName() {
```

please move this to `ViewCatalogTests` so that we can test the same behavior with different catalog implementations

Reply:
Done